tests/integration/conftest.py

```diff
@@ -0,0 +1,473 @@
+import os
+import pytest
+
+import subprocess as subp
+import yaml
+from packaging import version
+import time
+
+import pynetbox
+import requests
+import atexit
+import faker as _faker
+
+
+DOCKER_PROJECT_PREFIX = "pytest_pynetbox"
+DEVICETYPE_LIBRARY_OBJECTS = [
+    "A10/TH6430.yaml",
+    "APC/AP8868.yaml",
+    "Arista/DCS-7010T-48.yaml",
+    "Arista/DCS-7050TX3-48C8.yaml",
+    "Arista/DCS-7280CR3-32P4.yaml",
+    "Arista/DCS-7280SR-48C6.yaml",
+    "Dell/Networking_S4048-ON.yaml",
+    "Dell/PowerEdge_R640.yaml",
+    "Generic/48-port_copper_patch_panel.yaml",
+    "Generic/LC-48-port_fiber_patch_panel.yaml",
+    "Opengear/IM7248-2-DAC-US.yaml",
+]
```

Comment:
These are just the ones I thought to include to start us off. Feel free to add or remove, but in theory we could probably load the entire library without issue if we needed to.